internal/commands/push/command_test.go

```diff
@@ -437,14 +581,19 @@ An error occurred while uploading hosting assets: failed to remove /deleteme.htm
 
 				err := cmd.Handler(profile, ui, cli.Clients{Realm: realmClient})
 				assert.Equal(t, errors.New("2 error(s) occurred while importing hosting assets"), err)
-				assert.Equal(t, `Determining changes
```

Comment:
⚠️  drive-by here: the `404.html` and `index.html` lines output were non-deterministic (we run those actions asynchronously)... so loosened this assertion a bit to avoid the flakes